transformation_test/testdata/logging_processor-cassandra-gc/input.log

```diff
@@ -0,0 +1,43 @@
+INFO  [main] 2025-07-22 10:15:44,001 CassandraDaemon.java:687 - Starting Cassandra Server...
```

Comment:
This `input.log` file doesn't seem to follow the sample log format present in `cassandra_gc`. Could you please update the log file ?

https://github.com/GoogleCloudPlatform/ops-agent/blob/7298e03e3b4631e3a24a7e49bc61fe4b1d933d96/apps/cassandra.go#L154-L164

Reply:
Fixed and added some more edge cases here as well

Reply:
Thank you! There may be a need to run `make transformation_test_update` after the last update in https://github.com/GoogleCloudPlatform/ops-agent/pull/1999/commits/4f6880813d77011a010e42214734369c3334ba4b .